arangod/Aql/IResearchViewNode.cpp

```diff
@@ -2160,4 +2160,13 @@ IResearchViewNode::OptimizationState::replaceAllViewVariables(
   return uniqueVariables;
 }
 
+bool IResearchViewNode::isBuilding() const {
+  if (!_view || !ServerState::instance()->isCoordinator() ||
```

Comment:
"!ServerState::instance()->isCoordinator()"
I guess we should also report "is building" on the single server.